sas9.4-viya/scripts/compute_install.sh

```diff
@@ -103,24 +101,42 @@ else
     echo "ERROR: Failed to mount SASWork volume from Instance Store."
 fi
 
+#Cloning the sasinstall properties repo
+RETRIES=10
+DELAY=10
+COUNT=1
+while [ $COUNT -lt $RETRIES ]; do
+  git clone https://github.com/corecompete/sasinstalls.git sasinstalls
```

Comment:
all artifacts required for the sample must be included in the folder

Reply:
changed the code logic accordingly.